.github/workflows/appsec.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         runs-on: [ macos-12, macos-11, macos-10.15, ubuntu-22.04, ubuntu-20.04, ubuntu-18.04, windows-latest ]
-        go-version: [ 1.19, 1.18, 1.17 ]
+        go-version: [ 1.20, 1.19, 1.18 ]
```

Comment:
Based on the failed CI runs trying `1.2` instead of `1.20`, these need to be quoted so they're not treated as numbers. (Hooray for YAML!)